app/src/main/java/jp/kztproject/rewardedtodo/di/FragmentsTransitionManager.kt

```diff
@@ -4,23 +4,29 @@ import androidx.fragment.app.FragmentActivity
 import androidx.navigation.findNavController
 import jp.kztproject.rewardedtodo.R
 import jp.kztproject.rewardedtodo.presentation.reward.list.RewardListFragmentDirections
+import jp.kztproject.rewardedtodo.presentation.todo.TodoListFragmentDirections
 import project.seito.screen_transition.IFragmentsTransitionManager
 import javax.inject.Inject
 
 class FragmentsTransitionManager @Inject constructor() : IFragmentsTransitionManager {
-    override fun transitionToAuthFragment(activity: FragmentActivity?) {
-        activity?.findNavController(R.id.nav_host_fragment)?.navigate(RewardListFragmentDirections.actionRewardListFragmentToTodoistAuthFragment())
+
+    override fun transitionToTodoListFragment(activity: FragmentActivity) {
+        activity.findNavController(R.id.nav_host_fragment).navigate(RewardListFragmentDirections.toTodoListFragment())
+    }
+
+    override fun transitionToRewardListFragment(activity: FragmentActivity) {
+        activity.findNavController(R.id.nav_host_fragment).navigate(TodoListFragmentDirections.toRewardListFragment())
     }
 
-    override fun transitionToRewardDetailFragment(activity: FragmentActivity?) {
-        activity?.findNavController(R.id.nav_host_fragment)?.navigate(RewardListFragmentDirections.toRewardDetailFragment())
+    override fun transitionToSettingFragmentFromTodoListFragment(activity: FragmentActivity) {
+        activity.findNavController(R.id.nav_host_fragment).navigate(TodoListFragmentDirections.toTodoistAuthFragment())
     }
 
-    override fun transitionToRewardDetailFragment(activity: FragmentActivity?, rewardId: Int) {
-        activity?.findNavController(R.id.nav_host_fragment)?.navigate(RewardListFragmentDirections.toRewardDetailFragment(rewardId))
+    override fun transitionToSettingFragmentFromRewardListFragment(activity: FragmentActivity) {
+        activity.findNavController(R.id.nav_host_fragment).navigate(RewardListFragmentDirections.toTodoistAuthFragment())
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/cd1d2ea4f307b0df8c7e4d223455972a48ee2300/app/src/main/java/jp/kztproject/rewardedtodo/di/FragmentsTransitionManager.kt#L26"></span>Argument should be on a separate line (unless all arguments can fit a single line)</td>
    </tr>
  </tbody>
</table>